addons/native_dialog/gtk_filesel.c

```diff
@@ -46,6 +46,10 @@ static gboolean create_gtk_file_dialog(gpointer data)
 
    _al_gtk_make_transient(display, window);
 
```

Comment:
Personally I don't agree with the premise "should always use {} around an if block".  I find single statement blocks inelegant and unnecessary.  That's an stylistic issue.  In respect to regards like "it's error prone", my answer is: first, that's debatable; second I use Emacs, and its indentation is very clear e consistent; third, good C programmers should be very acquainted with that, making errors quite unlikely, not worth the wasteful non-elegant alternative.  Said that, of course, this is a pull request, so I should use the coding style of the host project.  What's the official convention for Allegro?